apps/events/views.py

```diff
@@ -630,3 +630,7 @@ def post(self, request, format=None):
         # All is clear, attendee is attended
         return Response({'message': (attendee.user.get_full_name() + ' er registrert som deltaker. Velkommen!'),
                          'attend_status': 10, 'attendee': attendee.id}, status=status.HTTP_200_OK)
+
+    @staticmethod
+    def get_extra_actions():
+        return []
```

Comment:
Is this for some strange reason absolutely needed for DRF 3.9?

Reply:
This has to do with DRF 3.8 adding the `get_extra_actions`-method to Views [(PR)](https://github.com/encode/django-rest-framework/pull/5705/files), [Changelog](https://www.django-rest-framework.org/community/release-notes/#38x-series), so what I think is happening here is that `AttendViewSet`, which inherits from `APIViewSet`, and not a generic viewset (viewsets.ViewSet), does not by default have the `get_extra_actions`-method on it, but we are for some reason using it as a "normal" viewset, _somewhere_, this _might_ be https://github.com/dotkom/onlineweb4/blob/358b15f1bc3ec25c8dd530b9186c6ab3a169beae/apps/events/urls.py#L38 and I am checking right now if adding `.as_view()` fixes the problem

Reply:
Update: nope, gives `AttributeError: 'function' object has no attribute 'get_extra_actions'
`